snow/engine/snowman/engine_test.go

```diff
@@ -3221,3 +3234,71 @@ func TestEngineAbortQueryWhenInPartition(t *testing.T) {
 
 	require.Contains(buff.String(), errInsufficientStake)
 }
+
+func TestEngineAcceptedHeight(t *testing.T) {
+	require := require.New(t)
+
+	engCfg := DefaultConfig(t)
+	engCfg.Params = snowball.Parameters{
+		K:                     2,
+		AlphaPreference:       2,
+		AlphaConfidence:       2,
+		Beta:                  1,
+		ConcurrentRepolls:     1,
+		OptimalProcessing:     1,
+		MaxOutstandingItems:   1,
+		MaxItemProcessingTime: 1,
+	}
+
+	vals := validators.NewManager()
+	engCfg.Validators = vals
+
+	_, _, _, vm, te := setup(t, engCfg)
+
+	vdr0 := ids.GenerateTestNodeID()
+	vdr1 := ids.GenerateTestNodeID()
+
+	require.NoError(vals.AddStaker(engCfg.Ctx.SubnetID, vdr0, nil, ids.Empty, 1))
+	require.NoError(vals.AddStaker(engCfg.Ctx.SubnetID, vdr1, nil, ids.Empty, 1))
+
+	blk1 := snowmantest.BuildChild(snowmantest.Genesis)
+
+	blk2 := snowmantest.BuildChild(blk1)
```

Comment:
Can we use `snowmantest.BuildDescendants` instead of calling `BuildChild` twice?